code/modules/mob/living/simple_animal/friendly/bat.dm

```diff
@@ -0,0 +1,42 @@
+/mob/living/simple_animal/bat
```

Comment:
Please avoid doing such copypasta and instead tell us why you aren't making the secbat a subtype of `/mob/living/simple_animal/hostile/retaliate/bat` so we can fix it.

Reply:
So it's not hostile.

Reply:
The `retaliate` subtype (which the original bats are part of) modifies said ""AI"" to be, as the name hints, more about retaliation than uncalled assault.
They'll go aggressive upon receiving damage from any source, rather than automatically upon spotting a mob they share no faction with.

Regardless, if you want to make sure these sec bats stay friendly and pick up no fight, the easiest and most boring way would be
```
/mob/living/simple_animal/hostile/retaliate/bat/secbat/Retaliate()
	return
```
You can try fancier solutions like petting or a prob() roll per tick to calm them, but that'll require half a dozen more lines or so.

Reply:
Ah, my concern was by creating a mob under the hostile category that wasn't hostile might cause problems.

Reply:
I'll move them across, but I'm still gonna make them gold core spawnable!

Reply:
The gold core spawnability is no issue.